.devcontainer/devcontainer.json

```diff
@@ -1,19 +1,31 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/python
 {
     "name": "Azure AI Search VoiceRAG",
-    "image": "mcr.microsoft.com/devcontainers/python:3.11",
+    "image": "mcr.microsoft.com/devcontainers/python:1-3.12-bullseye",
+    // Features to add to the dev container. More info: https://containers.dev/features.
     "features": {
+        "ghcr.io/devcontainers/features/azure-cli:1": {},
         "ghcr.io/devcontainers/features/node:1": {
             "version": "20"
         }
     },
+    // Configure tool-specific properties.
     "customizations": {
         "vscode": {
             "extensions": [
                 "esbenp.prettier-vscode",
-                "bradlc.vscode-tailwindcss"
+                "bradlc.vscode-tailwindcss",
+                "GitHub.copilot-chat"
             ]
         }
     },
-    "forwardPorts": [8765],
-    "remoteUser": "vscode"
-  }
\ No newline at end of file
+    // Use 'forwardPorts' to make a list of ports inside the container available locally.
+    "forwardPorts": [
+        8765
+    ],
+    // Use 'postCreateCommand' to run commands after the container is created.
+    "postCreateCommand": "cd app/frontend && npm install && cd ../backend && pip install -r requirements.txt"
+    // Uncomment to connect as root instead. More info: https://aka.ms/dev-containers-non-root.
```

Comment:
Suggestion: if a remoteUser isn't required, let's remove it to keep the file as simple as possible.

Reply:
Addressed in b4af6e5d4bf617ad521bdab516b1f9e842254029